api/routers/main.js

```diff
@@ -1,7 +1,8 @@
 const router = require('express').Router();
+const passport = require('../policies/passport');
 const MainController = require('../controllers/main');
 
-router.get('/', MainController.index);
-router.get('/sites(/*)?', MainController.index);
+router.get('/', passport, MainController.index);
+router.get('/sites(/*)?', passport, MainController.index);
```

Comment:
I'm actually wrong again. `req.isAuthenticated()` does exist and seems to do what we want. I was switching between projects and confused myself. Gack, many sorries!